services/iaasalpha/wait/wait.go

```diff
@@ -15,11 +15,14 @@ const (
 	AvailableStatus = "AVAILABLE"
 	DeleteSuccess   = "DELETED"
 	ErrorStatus     = "ERROR"
+	ActiveStatus    = "ACTIVE"
+	ResizingStatus  = "RESIZING"
```

Comment:
Are the two new ones only regarding Servers and the previous ones only regarding Volumes? If yes I think we could make it more clear by adding prefixes like `VolumeAvailableStatus`, `ServerActiveStatus`, etc

Reply:
Good point